packages/dashboard-frontend/src/components/WorkspaceProgress/StartingSteps/WorkspaceConditions/index.tsx

```diff
@@ -89,7 +89,7 @@ export default class StartingStepWorkspaceConditions extends ProgressStep<Props,
       this.timerId = window.setTimeout(() => {
         this.setState({
           subConditionTitle:
-            'Downloading IDE binaries... (it can take a few minutes depending on your internet connection)',
+            'Downloading IDE binaries... (it can take a few minutes depending on your connection)',
```

Comment:
I'd leave it as `it can take a few minutes`

Reply:
applied